scripts/generate-docs-markdown.js

```diff
@@ -188,7 +194,7 @@ section for more detail.
 
 If using the \`overrides\` key of the theme as documented
 [here](/customization/themes#customizing-all-instances-of-a-component-type),
-you need to use the following style sheet name: \`${styles.name}\`.`;
+you need to use the following style sheet name: \`${styles.name}\`.` : '';
```

Comment:
This is just the other side of the ternary that checks for styles.classes.length